src/dmtcp_coordinator.cpp

```diff
@@ -239,6 +240,14 @@ static pid_t _nextVirtualPid = INITIAL_VIRTUAL_PID;
 static int theNextClientNumber = 1;
 vector<CoordClient *>clients;
 
+static inline void
+ltrim(string &s)
+{
+  s.erase(s.begin(), std::find_if(s.begin(), s.end(), [](unsigned char ch) {
```

Comment:
Can we rewrite this as a standard iteration.  Making this a lambda function just helps to ensure that the majority of people with a B.S. in computer science will have to start googling to figure out what this code means.

And using a lambda function does not seem to add greater clarity.  When I think about trimming space from the beginning of a string, my first thought is not that i need to write an extra helper function in order to reduce the complexity of the task.  :-)